monitoring/prober/scd/test_constraint_simple.py

```diff
@@ -21,11 +21,6 @@
 CONSTRAINT_TYPE = register_resource_type(1, 'Single constraint')
 
 
-def test_set_test_owner_ids(test_owner):
```

Comment:
We'll need to sync once more from scd_api_update to grab #565 which will remove this block

Reply:
synced!